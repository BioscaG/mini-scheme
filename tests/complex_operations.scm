(define (main)
  (let ((a 10)
        (b 3))
    (display (+ (* a b) (/ a b)))
    (newline)
    (display (> (* a b) (/ a b)))
    (newline)))