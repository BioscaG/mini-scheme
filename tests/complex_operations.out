33.333333333333336
#t
